modules/c++/re/include/re/PCRE.h

```diff
@@ -29,10 +29,18 @@
 #define USE_PCRE
 #endif
 
+#include <sys/sys_config.h>
```

Comment:
So... the `re` module sounds like it's a generic regex module, but then it names the actual class that you interact with directly after the library that it uses for implementation.  This gets extra confusing now that we have an STL implementation.  How about renaming PCRE.h --> Regex.h (and rename class to `Regex`), PCRE.cpp --> RegexPCRE.cpp, PCRE_STL.cpp --> RegexSTL.cpp, PCREException --> RegexException.  Then rename unit tests and calling code appropriately.  Obviously the downside of this is this will ripple through all the calling code but I think it'll be named more correctly at that point.